llama-index-integrations/readers/llama-index-readers-google/llama_index/readers/google/sheets/base.py

```diff
@@ -79,6 +80,21 @@ def load_data(self, spreadsheet_ids: List[str]) -> List[Document]:
             )
         return results
 
+    def load_data_in_pandas(self, spreadsheet_ids: List[str]) -> List[pd.DataFrame]:
```

Comment:
Can we add some documentation for this at least? Probably a good time to update the README to reflect v0.10.x :)